actions/release/reset-draft/entrypoint/main.go

```diff
@@ -93,7 +93,17 @@ func main() {
 		fail(fmt.Errorf("unexpected response from delete draft release request: %s", dump))
 	}
 
-	fmt.Printf("::set-output name=current_version::%s\n", releases[0].TagName)
+	outputFileName, ok := os.LookupEnv("GITHUB_OUTPUT")
+	if !ok {
+		fail(errors.New("GITHUB_OUT is not set, see https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter"))
```

Comment:
Similar to above - I think this error message should read: `GITHUB_OUTPUT is not set`

Reply:
Also I'm probably fine not checking this `ok` condition - GitHub guarantees that this environment variable is set, and even if it's not set we will just fail on the next line, albeit with a less helpful error message.

I'm happy to leave this check in the code, but I'd be happy to see it removed either. I'm just pointing out the user experience is pretty much identical either way.

Reply:
I don't mind either, anyone else with a strong opinion on this?